GameServer/Logic/Game.cs

```diff
@@ -65,10 +65,23 @@ public bool Initialize(Address address, string baseKey)
 
             Blowfish = new BlowFish(key);
             PacketHandlerManager = new PacketHandlerManager(this);
-            _map = new SummonersRift(this);
+            switch(Config.GameConfig.Map)
```

Comment:
I think this could as well be implemented "properly", as we're going to need a dynamic system later on either way. So I'd suggest creating a dictionary with `<mapId, map>` pairs (could even go `<name, map>` though), a `RegisterMap` function and then initialize that with our current known maps (by using the `RegisterMap` function of course)